src/services/default-rule.js

```diff
@@ -47,13 +47,15 @@ module.exports = class DefaultRuleHelper {
 				value: bodyData.target_field,
 				model_names: { [Op.contains]: [modelName] },
 				required: true,
+				allow_filtering: true,
```

Comment:
@sumanvpacewisdom we are only validating the allow_filtering:true, for other entity types do we have validation ?